pkg/controller/installation/core_controller.go

```diff
@@ -718,10 +718,6 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 			r.status.SetDegraded("An error occurred retrieving the authentication configuration", err.Error())
 			return reconcile.Result{}, err
 		}
-		if authentication != nil && authentication.Status.State != operator.TigeraStatusReady {
```

Comment:
Was this something that shouldn't have been here in the first place? I'm just wondering how the current changes in for this PR prompted removing this.

Reply:
That's right. It's in the title of this PR but most of the discussion about it was had in dm.